src/npc.cpp

```diff
@@ -1064,6 +1067,31 @@ void npc::form_opinion( const player &u )
     add_msg( m_debug, "%s formed an opinion of u: %s", name, npc_attitude_id( attitude ) );
 }
 
+void npc::mutiny()
+{
+    if( !my_fac || !is_player_ally() ) {
+        return;
+    }
+    bool seen = g->u.sees( pos() );
+    if( seen ) {
+        add_msg( m_bad, _( "%s is tired of your incompetent leadership and abuse!" ), disp_name() );
+    }
+    // NPCs leaving your faction due to mistreatment further reduce their opinion of you
+    if( my_fac->likes_u < -10 ) {
+        op_of_u.trust += my_fac->respects_u / 10;
+        op_of_u.anger += my_fac->likes_u / 10;
+    }
+    // NPCs leaving your faction for abuse reduce the hatred your (remaining) followers
+    // feel for you, but also reduces their respect for you.
+    my_fac->likes_u = std::max( 0, my_fac->likes_u / 2 + 10 );
+    my_fac->respects_u -= 5;
+    set_fac( faction_id( "amf" ) );
+    say( _( "<follower_mutiny>  Adios, motherfucker!" ), sounds::sound_t::order );
```

Comment:
```suggestion
    say( _( "<follower_mutiny>  Adios, <name_b>!" ), sounds::sound_t::order );
```

Reply:
No, the entire (dubiously funny) joke is that the faction name is AMF because the last thing they say to you is "adios, motherfucker."